bitnami.libsonnet

```diff
@@ -71,14 +71,14 @@ local perCloudSvcSpec(cloud) = (
     },
     cm_dns_metadata:: {
       annotations+: {
-        "certmanager.k8s.io/cluster-issuer": "letsencrypt-prod-dns",
-        "certmanager.k8s.io/acme-challenge-type": "dns01",
-        "certmanager.k8s.io/acme-dns01-provider": "default",
+        "cert-manager.io/cluster-issuer": "letsencrypt-prod-dns",
```

Comment:
As I understand, only `cert-manager.io/cluster-issuer` new annotation needs to be present, as the old annotation selectors (acme-challenge-type and acme-dns01-provider)  are deprecated, as the cluster-issuer no has a specific `selector` field for that (~"server-side").

Reply:
I've got these changes on my local branch: pushing it to this PR